examples/graphs/proteins_pytorch_geometric/model_def.py

```diff
@@ -74,54 +73,43 @@ def __init__(self, context: PyTorchTrialContext):
 
         self.num_feature = self.dataset.num_features
         self.num_class = self.dataset.num_classes
-        average_nodes = int(self.dataset.data.x.size(0) / len(self.dataset))
 
         self.model = self.context.wrap_model(
             Net(
-                self.num_feature,
-                self.num_class,
-                self.context.get_hparam("hidden_channels"),
-                average_nodes,
+                self.dataset.num_features,
+                self.dataset.num_classes,
+                self.context.get_hparam("topk_pooling_ratio"),
+                self.context.get_hparam("dropout"),
             )
         )
 
         self.optimizer = self.context.wrap_optimizer(
             torch.optim.Adam(
                 self.model.parameters(),
                 lr=self.context.get_hparam("lr"),
-                weight_decay=self.context.get_hparam("weight_decay"),
             )
         )
 
     def train_batch(self, batch, epoch_idx: int, batch_idx: int):
```

Comment:
Should we add a type annotation here for batch?

Reply:
it starts conflicting with our suggested type in the harness code (pytorch.TorchData), which makes things look very red if you have a type checker.